### PR TITLE
Papercut fix for #75

### PR DESCRIPTION
**Pull request by lucernae**
Papercut fix pull request.
Mainly about styles.

### CODE REVIEW

File: deployment/docker-compose-dev.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#
+# Production server with UWSGI configured to run on port 8080
```
Comment (timlinux) [inline deployment/docker-compose-dev.yml]: dev server right?

**Comment by timlinux** [comment]
Hi @lucernae 

Thanks for this. The only thing I am not so keen on is using separate docker-compose-dev.yml - for me it breaks DRY since much of the content is boilerplate copy over from docker-compose.yml. I used to use this pattern too but switched to rather using a single yml file to keep things simple. With the $(PROJECT)-dev suffix in the makefile, it should ensure that the containers are properly namespaced. For dev environment you just need to sure you have a separate git checkout so that the pg etc dirs are not share between dev and production. Normally this would be the case anyway because for a single repo checkout, you cannot run it in both dev and prod state at the same time.

Regards

Tim

**Comment by lucernae** [comment]
@timlinux My problem is, since I want to expose the postgres port from docker to host in my dev env, I thought maybe I should make a different yml. I will try to find a workaround for this. Previously I created devdb configuration which is basically the same as db configuration but with exposed port. But that also doesn't follow DRY.

**Comment by timlinux** [comment]
Ok I am still working to get an optimal dev env with the docker stuff...for now I guess we have to do hacky things...but I think I would rather just add the port mapping to the master yaml right now

**Comment by lucernae** [comment]
I did a little bit research about extending yml using this configuration key: 
https://docs.docker.com/compose/extends/#example

But it seems not dry enough right now. I still have to redefine the links in production and development. So I guess we will revert it back first using one configuration for docker-compose.yml

File: deployment/docker-compose.yml [hunk 2]
```diff
@@ -56,63 +74,27 @@ web:
...
+# different port for the web.
 devdb:
-  image: kartoza/postgis
-  volumes:
-    - ./pg/postgres_data:/var/lib/postgresql
-  environment:
-    - USERNAME=docker
-    - PASS=docker
-  # RMN : Expose ports so we can access it from the host, maybe using pgAdmin
+  <<: *db
```
Comment (lucernae) [inline deployment/docker-compose.yml]: Check this out @timlinux . I uses yml anchor and references to simplify it. Perhaps it is better to use extend keyword, but it is not really DRY at the moment.

**Pull request merged**